ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/loopbacks-interfaces.yml

```diff
@@ -10,6 +10,7 @@ loopback_interfaces:
   - name: Loopback1
     description: VTEP_VXLAN_Tunnel_Source
     ip_address: 192.168.254.3/32
+    isis_bfd: true
```

Comment:
Please move this to Loopback99 where you have all the other isis knobs.

Reply:
Fixed